wishlist/app/com/commercetools/sunrise/wishlist/controllers/ClearWishlistControllerAction.java

```diff
@@ -9,6 +9,13 @@
 
 @ImplementedBy(DefaultClearWishlistControllerAction.class)
 public interface ClearWishlistControllerAction extends ControllerAction, Function<ShoppingList, CompletionStage<ShoppingList>> {
+    /**
+     * Removes all line items from the given wishlist.
+     *
+     * @param wishlist the wishlist
+     *
+     * @return the completion stage for this action
+     */
```

Comment:
As discussed, I would still use javdoc on the method because it provides more context. But I agree with your improved method javadoc and applied this style to all wishlist controller actions 👍